tests/pytorch/mock_sparse2/test_sparse_matrix.py

```diff
@@ -2,6 +2,8 @@
 import torch
 import sys
 
+import backend as F
```

Comment:
Why do we use backend here? Shall we directly use Pytorch?

Reply:
I think it is for device control L27.